routes/index.js

```diff
@@ -42,6 +42,7 @@ exports = module.exports = function(app) {
 	app.get('/about', routes.views.about);
 	app.get('/brewers', routes.views.brewers);
 	app.get('/beers', routes.views.beers);
+	app.get('/beerProfile', routes.views.beers);
```

Comment:
This beerProfile should accept a beer slug as a parameter, and present a profile page for that beer.